frontend/tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "noImplicitAny": false
```

Comment:
is there a particular reason you added this? i feel like its probably better to have stricter type checking?

Reply:
LOL i don't remember why so i removed it (: